js/index.js

```diff
@@ -13,4 +13,26 @@ for (const item of skills){
     skill.innerText = item;
     skillsList.appendChild(skill);
 }
-    
+let messageSection = document.getElementById('messages');
+let messageList = messageSection.querySelector('ul');
+let newMessage = document.createElement('li');
+newMessage.getElementByName('usersName').innerHTML = "<a href=mailto: + newMessage.getElementByName('usersEmail').email>";
+removeButton.innerText = 'remove';
+removeButton.Type = 'button';
+removeButton.addEventListener('click', () => {
+    let entry = document.getElementByType('button').parentNode.nodeName;
+    entry.remove();
+    newMessage.appendChild(removeButton);
+    messageList.appendChild(newMessage);
```

Comment:
The remove button needs to be created inside the submit event handler and is appended to each new message.